app/jupyter-lite.json

```diff
@@ -11,6 +11,7 @@
     "fullMathjaxUrl": "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js",
     "fullStaticUrl": "./build",
     "licensesUrl": "./lab/api/licenses",
-    "mathjaxConfig": "TeX-AMS_CHTML-full,Safe"
+    "mathjaxConfig": "TeX-AMS_CHTML-full,Safe",
+    "preferredPath": "/"
```

Comment:
This will be removed when https://github.com/jupyterlab/jupyterlab/pull/12521 is available in a lab release.

Reply:
Tracked in https://github.com/jupyterlite/jupyterlite/issues/630